contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -447,6 +445,25 @@ private void checkIfQuotaExceeded(Long diskQuotaSizeInKB,
         }
     }
 
+    private boolean retryIsFile(FileSystem fs, Path rootPath) {
+        for (int attempt = 1; attempt <= getMaxAttempts(); attempt++) {
+            try {
+                return fs.isFile(rootPath);
+            } catch (IOException e) {
+                logger.error("Error while calling isFile for path:" + rootPath.toString());
```

Comment:
I will add the current attempt and max attempt information in the error message.